templates/basic/server.js

```diff
@@ -1,5 +1,4 @@
-import express from "express";
-
+const express = require("express");
```

Comment:
Please follow `ES6`. It's a standard we're following across templates.
Don't change to `CommonJS`.

Reply:
ok understood I don't know y i changed this

Reply:
It throws the same error when testing as jest follows common js by default but Iam not sure why it throws error from this file even though it is not a test file and the app is exported correctly 
![image](https://github.com/user-attachments/assets/0a273539-495d-4d0b-9ec8-6b064a56c162)
I can only change this once I know what to do in the server test file as it imports  from this file

Reply:
Hey I'll push a commit to fix this now.